templates/agent-conf.d/network.yaml.erb

```diff
@@ -14,3 +15,47 @@ instances:
     excluded_interface_re: <%= @excluded_interface_re %>
 <% end -%>
     combine_connection_states: <%= @combine_connection_states %>
+    collect_rate_metrics: <%= @collect_rate_metrics %>
+    collect_count_metrics: <%= @collect_count_metrics %>
+    conntrack_path: <%= @conntrack_path %>
+    use_sudo_conntrack: <%= @use_sudo_conntrack %>
+<% if !@whitelist_conntrack_metrics.empty? -%>
+    whitelist_conntrack_metrics:
+<%- (@whitelist_conntrack_metrics.each do |metric| -%>
+      - <%= metric %>
+<% end ) -%>
+<% end -%>
+<% if !@blacklist_conntrack_metrics.empty? -%>
+    blacklist_conntrack_metrics:
+<%- (@blacklist_conntrack_metrics.each do |metric| -%>
+      - <%= metric %>
+<% end ) -%>
+<% end -%>
+    collect_aws_ena_metrics: <%= @collect_aws_ena_metrics %>
+    collect_ethtool_metrics: <%= @collect_ethtool_metrics %>
+<% if !@tags.empty? -%>
+    tags:
+<%- (@tags.each do |tag| -%>
+      - <%= tag %>
+<% end ) -%>
+<% end -%>
+<% if !@service.empty? -%>
+    service: <%= @service %>
+<% end -%>
```

Comment:
`service` key should live under `init_config`, not under `instances`.

Reply:
https://github.com/DataDog/integrations-core/blob/ba25dd5cfc9f6769435d60bdd2b92d7c5f45bee1/network/datadog_checks/network/data/conf.yaml.default#L135

Reply:
You're right, please disregard this part.